packages/flutter/lib/src/material/spell_check_suggestions_toolbar.dart

```diff
@@ -0,0 +1,208 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/services.dart' show SuggestionSpan;
+import 'package:flutter/widgets.dart';
+
+import 'colors.dart';
+import 'material.dart';
+import 'material_localizations.dart';
+import 'spell_check_suggestions_toolbar_layout_delegate.dart';
+import 'text_selection_toolbar_text_button.dart';
+
+// Minimal padding from all edges of the selection toolbar to all edges of the
+// viewport.
+const double _kToolbarScreenPadding = 8.0;
+const double _kHandleSize = 22.0;
+
+// Padding between the toolbar and the anchor.
+const double _kToolbarContentDistanceBelow = _kHandleSize - 3.0;
+
+// The default height of the MaterialSpellCheckSuggestionsToolbar, which
+// assumes there are the maximum number of spell check suggestions available, 3.
+const double _kDefaultToolbarHeight = 193.0;
```

Comment:
For all of these constant values here, would it be helpful to include where you got them in the comments?  Like "eyeballed on Android 13" or whatever.